upstairs/src/test.rs

```diff
@@ -506,11 +499,11 @@ mod up_test {
         assert_eq!(ds.completed.len(), 1);
     }
 
-    #[test]
-    fn work_flush_one_error_then_ok() {
+    #[tokio::test]
```

Comment:
I wonder if there is any impact to all these tests being inside tokio now.  I don't think there is really and choice though.

Reply:
right, yeah. they all require async now.